tools/needs_binaries.sh

```diff
@@ -7,7 +7,7 @@ cd ../packages
 arch=$(uname -m)
 exclusions="cabal.rb cros_resize.rb docx2txt.rb dpkg.rb fpc.rb freedos.rb ghc.rb kr.rb"
 exclusions+=" libtinfo.rb lsb_release.rb mysqltuner.rb nodebrew.rb nvm.rb oc.rb stack.rb"
-exclusions+=" txt2regex.rb uwsgi.rb xdg_base.rb"
+exclusions+=" txt2regex.rb uwsgi.rb xdg_base.rb docbook.rb docbook_xsl.rb docbook_xml.rb"
```

Comment:
We don't want to compile these packages everytime they need installed.  That doesn't make sense.

Reply:
What does this file do anyway? I thought it builds binaries.

Reply:
What you are doing is excluding docbook packages from the binary builds.  We don't want to do that. :)

Reply:
Yes, we do! I could make a workaround, if you are willing to put it in the postinstall.

Reply:
I reverted this.

Reply:
BTW, it doesn't 'compile' them, per se. It copies the docbook files to a directory and then creates a catalog.